### PR TITLE
Default requests to addons-frontend and fall back to addons-server.

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Fixes https://github.com/mozilla/addons-server/issues/9080

Todo:

* [x] At least optimize API access for performance, default `/api/v3|v4/` requests to hit olympia directly

### CODE REVIEW

**Comment by EnTeQuAk** [comment]
r? @autrilla maybe?

This is a very naive approach, am I missing something?

File: addons.conf [hunk 1]
```diff
@@ -29,9 +35,59 @@ server {
...
+
     location / {
-        proxy_pass http://web/;
+        try_files $uri @frontendamo;
+    }
+
+    location @olympia {
+        # We are using uwsgi in our dev, stage and prod environments
+        # but not yet for our local development. So let's simply do a
+        # proxa_pass here to redirect to the django runserver.
```
Comment (autrilla) [inline addons.conf]: uber-nit: proxy_pass typo

**Review by autrilla — APPROVED** [review]
Looks good. Have you tested this somehow? We'll test in dev anyway, but my brain isn't particularly great at validating nginx configs.

**Comment by EnTeQuAk** [comment]
> Looks good. Have you tested this somehow?

Yeah, I'll try to build this locally, push to dockerhub and then have mat and eviljeff test this locally too and see if that works. I tested this for me locally and it works nicely so far.

> We'll test in dev anyway, but my brain isn't particularly great at validating nginx configs.

This won't land on -dev, -stage or so anyway but is 100% for local development

**Pull request merged**